core/src/runtime/mod.rs

```diff
@@ -37,14 +37,35 @@ use tokio_net::driver;
 use tokio_net::signal::unix::{self, SignalKind};
 use tokio_timer::{timer, Interval};
 
-/// Supported Unix signals.
+/// Supported [`Unix signals`].
+///
+/// [`Unix signals`]: https://dsa.cs.tsinghua.edu.cn/oj/static/unix_signal.html
```

Comment:
sorry, can't find any official spec for this. let's link to wikipedia instead. likely more permanent. https://en.wikipedia.org/wiki/Signal_(IPC)#POSIX_signals

Reply:
yeah, i was gonna do that. 👍